lib/takeaway.rb

```diff
@@ -0,0 +1,14 @@
+class Takeaway
+
+  def initialize(menu = @menu)
+    @menu = menu
+  end
+
+  def list_menu
```

Comment:
The list_menu method you have could remain in place and still print out the list of dishes.